MovingBlocks/Terasology
# Update Ukrainian locale with current strings

### Contains

Updates the `menu_uk` localization to translate all the current strings from the engine (227 new ones!)

### How to test

Launch the game, switch the language to Ukrainian, then check out various UI screens to see if things look alright. Maybe look things over with Google Translate :)

### Outstanding before merging

None!
